README.md

```diff
@@ -8,15 +8,27 @@
 # aries-framework-go
 
 ## Introduction
-A go framework for Aries
+A Go framework for Aries
 
-## Contributing
+## Build
+##### Prerequisites
+- Go 1.12
+- Docker
```

Comment:
Docker and Make are for running tests. Note: openssl is also used to generate test artifacts.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/pull/91/files#diff-04c6e90faac2675aa89e2176d2eec7d8R14) the subheading.